.gitignore

```diff
@@ -8,6 +8,7 @@ out/
 log/*.log
 tmp/**
 node_modules/
+package-lock.json
```

Comment:
checking lock files into version control actually helps when you need to [audit the code](https://docs.npmjs.com/auditing-package-dependencies-for-security-vulnerabilities).

Reply:
@jgravois that came over from reveal.js, but I agree, i'll add it in.